### PR TITLE
Update hub-to-hub-association.md

### PR DESCRIPTION
**Pull request by Marweis**
Added note:  Web parts configured to show content from all sites in the hub, will continue to show data from the original parent hub only

### CODE REVIEW

File: SharePoint/SharePointOnline/hub-to-hub-association.md [hunk 1]
```diff
@@ -36,6 +36,7 @@ Associate a [SharePoint hub](https://support.microsoft.com/office/what-is-a-shar
 >
 > - You must have administrator editing privileges to associate hubs to each other in the SharePoint admin center.
 > - Hub level associations do not change site permissions.
+> - Web parts configured to show content from all sites in the hub, will continue to show data from the original parent hub only
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/hub-to-hub-association.md:39]:
```suggestion
> - Web parts configured to show content from all sites in the hub will continue to show data from the original parent hub only.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @Marweis, kindly commit the suggested changes. Many thanks.

**Comment by Marweis** [comment]
@JohanFreelancer9 - committed suggestions. Please publish.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@MattEEvans 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Review by MattEEvans — APPROVED** [review]
Looks good. Thanks.

**Pull request merged**